custom/mvp_ucr/reports/data_sources/va_datasource.json

```diff
@@ -0,0 +1,303 @@
+                                        "q1004"
+                                    ]
+                                },
+                                "operator": "eq",
+                                "property_value": 1,
+                                "type": "boolean_expression"
+                            },
+                            "type": "conditional"
+                        },
+                        "expression_if_true": {
+                          "constant": "Yes",
+                          "type": "constant"
+                        },
+                        "test": {
+                            "expression": {
+                                "type": "property_path",
+                                "property_path": [
+                                    "form",
+                                    "q201-1114",
+                                    "q1001-1011",
+                                    "q1004"
+                                ]
+                            },
+                            "operator": "eq",
+                            "property_value": 1,
+                            "type": "boolean_expression"
+                        },
+                        "type": "conditional"
+                    },
+                    "expression_if_true": {
+                      "constant": "Yes",
+                      "type": "constant"
+                    },
+                    "test": {
+                        "expression": {
+                            "type": "property_path",
+                            "property_path": [
+                                "form",
+                                "interview",
+                                "q1004"
+                            ]
+                        },
+                        "operator": "eq",
+                        "property_value": 1,
+                        "type": "boolean_expression"
+                    },
+                    "type": "conditional"
+                  },
+                  "is_nullable": true,
+                  "is_primary_key": false,
+                  "transform": {
+                  },
+                  "type": "expression"
+            },
+            {
+                "column_id": "death_place",
+                "datatype": "string",
+                "display_name": "Death Place",
+                "expression": {
+                    "type": "mvp_death_place"
+                },
+                "is_nullable": true,
+                "is_primary_key": false,
+                "transform": {},
+                "type": "expression"
+            },
+            {
+                "column_id": "medical_reason",
+                "datatype": "string",
+                "display_name": "Medical Reason",
+                "expression": {
+                    "datatype": null,
+                    "property_name": "mvp_medical_cause",
+                    "type": "property_name"
+                },
```

Comment:
Looks like maybe this expression is just supposed to be:
```json
"expression": {
    "type": "mvp_medical_cause"
}
```
I haven't tried to run the report or anything though.